monitor/index.js

```diff
@@ -77,10 +77,10 @@ app.get('/eventsStats', async (req, res, next) => {
   try {
     const results = await readFile('./responses/eventsStats.json')
     results.ok =
-      results.onlyInHomeDeposits.length === 0 &&
-      results.onlyInForeignDeposits.length === 0 &&
-      results.onlyInHomeWithdrawals.length === 0 &&
-      results.onlyInForeignWithdrawals.length === 0
+      (results.onlyInHomeDeposits || results.home.deliveredMsgNotProcessedInForeign).length === 0 &&
```

Comment:
could you clarify why these changes are needed now?

Reply:
This was a bug found while adding the e2e tests. For AMB `results.onlyInHomeDeposits` does not exists, so `results.onlyInHomeDeposits.length` was throwing an error

Reply:
thanks